packages/zen-bull-vault/src/ZenEmergencyShutdown.sol

```diff
@@ -1,6 +1,5 @@
-// SPDX-License-Identifier: GPL-3.0-only
+//SPDX-License-Identifier: BUSL-1.1
```

Comment:
`StrategyMath.sol` is imported here which is `//SPDX-License-Identifier: AGPL-3.0-only `, doesn't this mean we have to make it GPL?

We also import our contract `UniFlash.sol`, which may need to be GPL, see above.